core-api/src/main/java/com/optimizely/ab/bucketing/Bucketer.java

```diff
@@ -125,12 +125,14 @@ private Variation bucketToVariation(@Nonnull Experiment experiment,
 
     /**
      * Assign a {@link Variation} of an {@link Experiment} to a user based on hashed value from murmurhash3.
-     * @param experiment The Experiment in which the user is to be bucketed.
+     *
+     * @param experiment  The Experiment in which the user is to be bucketed.
      * @param bucketingId string A customer-assigned value used to create the key for the murmur hash.
      * @return Variation the user is bucketed into or null.
      */
-    public @Nullable Variation bucket(@Nonnull Experiment experiment,
-                                      @Nonnull String bucketingId) {
+    public @Nullable
+    Variation bucket(@Nonnull Experiment experiment,
```

Comment:
Same as other comments about annotation placement for return types. This is formatting is worse.